### PR TITLE
setup: pin cylc-flow version

### PR DESCRIPTION
**Pull request by oliver-sanders**
* See https://github.com/cylc/cylc-admin/issues/130

### CODE REVIEW

**Comment by oliver-sanders** [comment]
Note, metomi-rose does not presently use the `.dev` convention.

**Review by MetRonnie — COMMENTED** [review]
Do we not want to pin the metomi-rose version?

File: setup.py [hunk 1]
```diff
@@ -28,7 +28,7 @@
 
 INSTALL_REQUIRES = [
     'metomi-rose>=2.0b2',
```
Comment (oliver-sanders) [inline setup.py:30]:
```suggestion
    'metomi-rose==2.0b2',
```

**Comment by oliver-sanders** [comment]
No reason, but will have to re do this again at release time anyway.

**Review by MetRonnie — APPROVED** [review]

**Comment by MetRonnie** [comment]
Oh I see, because it's going to be 2.0b3?

**Pull request merged**